code/modules/reagents/chemistry/reagents.dm

```diff
@@ -62,6 +62,8 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	var/evaporation_rate = 0.5
 	///if this reagent should condense down into a liquid
 	var/condenses_liquid = TRUE
+	///does this reagent convert into a gas
+	var/aerolizes = FALSE
```

Comment:
Misspelling.
Should be something more like "liquid_evaporation"

Reply:
its technically right chem gases are just  an aerosol agent

Reply:
forgot the so in their nvm